src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    @Override
+    public String toString() {
+        return super.toString();
```

Comment:
add a meaningful custom implementation
don't forget to call .name() on Color to get its string representation (more on this - in the [checklist ](https://github.com/mate-academy/jv-lottery/blob/master/checklist.md))